cmd/oras/internal/option/pusher.go

```diff
@@ -73,3 +98,22 @@ func decodeJSON(filename string, v interface{}) error {
 	defer file.Close()
 	return json.NewDecoder(file).Decode(v)
 }
+
+// getAnnotationsMap resharps annotationslice to k-v type and updates annotations
+func getAnnotationsMap(ManifestAnnotations []string, annotations map[string]map[string]string) error {
+	re := regexp.MustCompile(`=\s*`)
```

Comment:
Is there any reason that we can only use regular expression but not string process functions like `string.SplitN`?

Also this expression don't need to get compiled every time `getAnnotationsMap` is called.

Reply:
fixed